editor/debugger/editor_profiler.cpp

```diff
@@ -362,6 +366,9 @@ void EditorProfiler::_update_frame() {
 			item->set_tooltip_text(0, it.name + "\n" + it.script + ":" + itos(it.line));
 
 			float time = dtime == DISPLAY_SELF_TIME ? it.self : it.total;
+			if (!display_internal_profiles->is_pressed() && dtime == DISPLAY_SELF_TIME) {
```

Comment:
```suggestion
			if (dtime == DISPLAY_SELF_TIME && !display_internal_profiles->is_pressed()) {
```
Put cheaper condition first